src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -2,14 +2,14 @@
 #include "JEventProcessorPODIO.h"
 
 #include <JANA/JApplication.h>
-#include <JANA/JLogger.h>
 #include <JANA/Services/JParameterManager.h>
 #include <JANA/Utils/JTypeInfo.h>
 #include <fmt/core.h>
 #include <podio/CollectionBase.h>
 #include <podio/Frame.h>
 #include <spdlog/common.h>
 #include <exception>
+#include <iosfwd>
```

Comment:
@wdconinc yet another IWYU loop:
```diff
diff --git a/src/services/io/podio/JEventProcessorPODIO.cc b/src/services/io/podio/JEventProcessorPODIO.cc
index 0a27536..b4ee1c7 100644
--- a/src/services/io/podio/JEventProcessorPODIO.cc
+++ b/src/services/io/podio/JEventProcessorPODIO.cc
@@ -2,6 +2,7 @@
 #include "JEventProcessorPODIO.h"
 
 #include <JANA/JApplication.h>
+#include <JANA/JLogger.h>
 #include <JANA/Services/JParameterManager.h>
 #include <JANA/Utils/JTypeInfo.h>
 #include <fmt/core.h>
@@ -9,7 +10,6 @@
 #include <podio/Frame.h>
 #include <spdlog/common.h>
 #include <exception>
-#include <iosfwd>
 
 #include "services/log/Log_service.h"
```

Reply:
Is this a loop? It seems the clang-tidy-iwyu job completed in less than the timeout. I'm not going to claim to understand exactly why it suggests what it suggests all the time, but sometimes it seems merging main commits in a branch causes it to change its mind.

Reply:
Sure, I can assume that the new suggestion is up-to-date.